locales/en/moves.json

```diff
@@ -184,5 +184,34 @@
       "personal_details": "personal details",
       "risk": "risk information"
     }
+  },
+  "update_flash": {
+    "supplier_message": "This move has been updated with the supplier",
```

Comment:
I would actually consider dropping this message. It's not _strictly_ true as I'm not sure we notify them of these changes yet. They would have to be subscribed to the webhooks.

Also, we removed that extra copy from the other banners so I feel we could potentially drop this paragraph altogether and just go with the heading. See what @Andy-Millar thinks though.

Reply:
These are the messages that @Andy-Millar and @jason-sims supplied

Reply:
For the users benefit we are saying the supplier has received the changes (via email notifications & later webhooks). Can leave copy in for now and will raise separate ticket to change if needed.